app/src/main/java/me/rocka/fcitx5test/ui/common/DialogSeekBarPreference.kt

```diff
@@ -29,6 +29,9 @@ class DialogSeekBarPreference : Preference {
     var step: Int = 1
     var unit: String = ""
 
+    private val currentValue: Int =
+        preferenceDataStore?.getInt(key, defaultValue) ?: defaultValue
```

Comment:
You may want a getter here:

```suggestion
    private val currentValue: Int
        get() = preferenceDataStore?.getInt(key, defaultValue) ?: defaultValue
```

If you make `currentValue` a property, it's value cannot be changed after `DialogSeekBarPreference` being consturcted.